lib/Graph.dart

```diff
@@ -163,6 +163,8 @@ class Node {
 
   Offset position = Offset(0, 0);
 
+  String lineType = 'default';
```

Comment:
Great work, I think an enumType would be more preferred

Reply:
Hi!
Sorry for late response, I added the enum for lineType as requested